### PR TITLE
metronom added to Node clients

### PR DESCRIPTION
**Pull request by saracalihan**
Metronom is user friendly Redis ORM based on [node-redis](https://github.com/redis/node-redis)

You can **save**, **read**, **update**, **filter**, **delete** and **bulk** operations JavaScript objects in Redis **without needing to know Redis commands**.

It is used effortlessly without installing any plugins like RedisJSON. The system works with Hashes. It shreds the objects and saves them as key strings in the hash, and while reading, they break it down again according to the given scheme and type conversion with TypeScript.

**Source Code**: [anchovycation/metronom](https://github.com/anchovycation/metronom)
**NPM Package**: [https://www.npmjs.com/package/metronom](https://www.npmjs.com/package/metronom)

### CODE REVIEW

**Comment by saracalihan** [comment]
@itamarhaber ready for review. Would you review it?

File: clients/nodejs/github.com/anchovycation/metronom.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "metronom",
+    "description": "User friendly Redis ORM for Node.js with asynchronous and TypeScript support.",
+    "homepage": "https://anchovycation.github.io/metronom/",
+    "recommended": true,
```
Comment (itamarhaber) [inline clients/nodejs/github.com/anchovycation/metronom.json:5]: This attribute is reserved for the community.

```suggestion
```

**Review by itamarhaber — CHANGES_REQUESTED** [review]
Howdy @saracalihan and congrats on making this client :)

Please address the single comment and I'll happily merge (sorry for initial delay)

File: clients/nodejs/github.com/anchovycation/metronom.json (hunk 1 above)
Comment (saracalihan) [reply clients/nodejs/github.com/anchovycation/metronom.json]: @itamarhaber  I made the correction you requested, thank you for your interest

**Comment by saracalihan** [comment]
@itamarhaber  I made the correction you requested, thank you for your interest

**Comment by saracalihan** [comment]
@itamarhaber Would you review it again

**Review by itamarhaber — APPROVED** [review]

**Pull request merged**